main.tf

```diff
@@ -1,13 +1,13 @@
 resource "google_container_cluster" "primary" {
   name = "${var.name}"
 
-  zone  = "${var.region}-${var.zones[0]}"
-  count = "${var.regional_cluster ||  var.nat_enabled ? 0 : 1 }"
+  location = "${var.region}-${var.zones[0]}"
+  count    = "${var.regional_cluster ||  var.nat_enabled ? 0 : 1 }"
 
   min_master_version = "${var.min_master_version}"
   enable_legacy_abac = false
 
-  network    = "${var.network == "" ? terraform.workspace : var.network}"
+  network    = "${var.network == "" ? local.network_name : var.network}"
```

Comment:
Pls move `locals` section to the beginning of this file (best practise)

Reply:
👍